src/Magnum/Math/Geometry/Distance.h

```diff
@@ -162,6 +162,34 @@ class Distance {
          * the square root.
          */
         template<class T> static T lineSegmentPointSquared(const Vector3<T>& a, const Vector3<T>& b, const Vector3<T>& point);
+
+        /**
+         * @brief Distance of point from plane, scaled by the length of the planes normal.
+         * @param point the point
+         * @param plane the plane
+         * @returns a value `> 0` if point is on positive half space,
+         *                  `< 0` if point is on negative half space,
+         *                  `0` if point is on the plane.
+         *
+         * More efficient than @ref pointPlane() when the planes normal is already
+         * normalized or when testing on which half space the point lies.
+         */
+        template<class T> static T pointPlane(const Vector3<T>& point, const Vector4<T>& plane) {
+            return (plane.xyz()*point).sum() + plane.w();
```

Comment:
In some cases you do not need normalized values, e.g. when just trying to find out whether the point is in front or behind the plane. Hence, forcing normalization here would prohibit that use case.